third_party/libvpx/BUILD.gn

```diff
@@ -127,10 +131,8 @@ if (current_cpu == "x86" || (current_cpu == "x64" && !is_msan)) {
   # symbols.
 
   source_set("libvpx_intrinsics_mmx") {
-    if (!use_cobalt_customizations) {
-      configs -= [ "//build/config/compiler:chromium_code" ]
-      configs += [ "//build/config/compiler:no_chromium_code" ]
-    }
+    configs -= [ "//build/config/compiler:chromium_code" ]
```

Comment:
Do we still need to guard this with `use_cobalt_customizations`?

Reply:
No.